apiserver/v1/types_apirequestcount.go

```diff
@@ -116,6 +116,13 @@ type PerUserAPIRequestCount struct {
 	// +kubebuilder:validation:MaxLength=512
 	UserName string `json:"username"`
 
+	// userAgent that made the request.
+	// The same user often has multiple binaries which connect (pods with many containers).  The different binaries
+	// will have different userAgents, but the same user.  In addition, we have userAgents with version information
+	// embedded and the userName isn't likely to change.
+	// +kubebuilder:validation:MaxLength=1024
+	UserAgent string `json:"userAgent"`
```

Comment:
add the markers for server-side-apply array type with multiple keys

Reply:
> add the markers for server-side-apply array type with multiple keys

I don't know them.  What are they?